<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enchain</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>720</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>540</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>540</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>25</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOnline_Help"/>
    <addaction name="actionAuthor_Info"/>
    <addaction name="actionCheck_Update"/>
    <addaction name="actionInfo"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionRename"/>
    <addaction name="actionCutImage"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFolder"/>
    <addaction name="actionOpenVideo"/>
    <addaction name="actionOpenImage"/>
    <addaction name="actionSaveImage"/>
    <addaction name="actionClose"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPipeline">
    <property name="title">
     <string>Pipeline</string>
    </property>
    <addaction name="actionSetUserInfo"/>
    <addaction name="actionDatasetInput"/>
   </widget>
   <addaction name="menuPipeline"/>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAuthor_Info">
   <property name="text">
    <string>Author Info</string>
   </property>
  </action>
  <action name="actionAbout_Enchain">
   <property name="text">
    <string>About Enchain</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>OpenFolder</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOnline_Help">
   <property name="text">
    <string>Online Help</string>
   </property>
  </action>
  <action name="actionCheck_Update">
   <property name="text">
    <string>Check Update</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="text">
    <string>OpenFolder</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpenVideo">
   <property name="text">
    <string>OpenVideo</string>
   </property>
  </action>
  <action name="actionOpenImage">
   <property name="text">
    <string>OpenImage</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>SaveImage</string>
   </property>
  </action>
  <action name="actionCutImage">
   <property name="text">
    <string>CutImage</string>
   </property>
  </action>
  <action name="actionDatasetInput">
   <property name="text">
    <string>DatasetInput</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSetUserInfo">
   <property name="text">
    <string>SetUserInfo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
